draft-ietf-acme-acme.md

```diff
@@ -662,10 +660,10 @@ contact (optional, array of string):
 related to this authorization. For example, the server may wish to notify the
 client about server-initiated revocation.
 
-agreement (optional, string):
-: A URI referring to a subscriber agreement or terms of service provided by the
-server (see below).  Including this field indicates the client's agreement with
-the referenced terms.
+terms-of-service-agreed (optional, boolean):
+: Including this field in a new-registration request, with a value of true,
+indicates the client's agreement with the terms of service. This field is not
+updateable by the client.
```

Comment:
This isn't really true, because clients might need to re-agree.  (And we don't want them to have to re-register.)  How about the following?

```
Once set to "true" by the client, the value of this field cannot be changed by the client.  If the server requires the client to refresh its agreement to the terms, it does so by clearing this field and accepting a registration update to reset it to "true".
```